src/amo/components/LandingPage.js

```diff
@@ -108,9 +112,43 @@ export class LandingPageBase extends React.Component {
     }
 
     const { addonType, html } = content;
+    const themeText = i18n.gettext(
+      oneLine`Change your browser's appearance. Choose from thousands of themes to give Firefox the
```

Comment:
We no longer use templates string tags in `i18n.gettext`, so we can remove `oneLine` here and anywhere else.